### PR TITLE
blog: add June 2022 newsletter

### PR DESCRIPTION
**Pull request by CollierCZ**

### CODE REVIEW

File: docs/blog/newsletter-june-2022.rst [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+*Interested in sponsoring the newsletter? Take a look at our* `sponsorship prospectus </sponsorship/newsletter/>`__.
+
+------------------
+Featured job posts
+------------------
+
+- `Technical Writer <https://jobs.writethedocs.org/job/718/technical-writer/>`__, Wix (Vilnius, Lithuania )
+- `Technical Writer (Developer Documentation) <https://jobs.writethedocs.org/job/717/technical-writer-developer-documentation-fully-remote/>`__,  Humanitec GmbH (Remote)
+-  `Technical Writer, Serverless, Google Cloud <https://jobs.writethedocs.org/job/731/technical-writer-serverless-google-cloud/>`__, Google (Kitchener, ON, Canada)
+- `Technical Writer (IntelliJ) <https://jobs.writethedocs.org/job/735/technical-writer-intellij/>`__, JetBrains (arious locations)
```
Comment (Bradamant3) [inline docs/blog/newsletter-june-2022.rst]:
```suggestion
- `Technical Writer (IntelliJ) <https://jobs.writethedocs.org/job/735/technical-writer-intellij/>`__, JetBrains (various locations)
```

File: docs/blog/newsletter-june-2022.rst [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+- `Developer Educator - Remote APJ <https://jobs.writethedocs.org/job/725/developer-educator-remote-apj/>`__, Twilio (Remote)
+- `Developer Evangelist <https://jobs.writethedocs.org/job/726/developer-evangelist/>`__, Twilio (Remote)
+
+*To apply for these jobs and more, visit the* `Write the Docs job board <https://jobs.writethedocs.org/>`_.
+
+------------------------
+Virtual events coming up
+------------------------
+
+- 02 June, 19:00 PDT (Los Angelos) - `June: Catch up and discussion <https://www.meetup.com/Write-the-Docs-LA/events/285814451/>`__
```
Comment (Bradamant3) [inline docs/blog/newsletter-june-2022.rst]: Depending on when we ship, maybe remove this line?

I think there's a Florida morning social coming up too, although I don't know how exhaustive we're being with this list.

File: docs/blog/newsletter-june-2022.rst [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+            <a href="https://www.LINK.COM">
+              <img style="margin-left: 15px;" alt="SPONSOR" src="/_static/img/sponsors/rtd.png">
+            </a>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+    <hr>
+
+*Interested in sponsoring the newsletter? Take a look at our* `sponsorship prospectus </sponsorship/newsletter/>`__.
```
Comment (ericholscher) [inline docs/blog/newsletter-june-2022.rst:100]: We can cut this for now, until we have a sponsor, or ship it without one.
```suggestion
```

File: docs/blog/newsletter-june-2022.rst [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+I certainly have learned a lot from her already on how to make a newsletter useful and interesting. But it's also important to recognize that the heart of the newsletter comes from you, the Write the Docs community. I had a conversation in Slack recently with a long-standing member who didn't realize they could suggest stories to be included.
+
+So as I start this journey, I'd like to (re-)invite you to come with me. If you see a conversation in Slack you think would be great for the newsletter, tag it with the ``:suggest-for-newsletter:`` emoji. We'll round up the most interesting and thought-provoking ideas and bring them to you.
+
+The past month was full of goodness outside the newsletter as we wrapped up the Portland conference. Those who were there talked a lot about how great all the talks and unconference sessions were. For those of us who couldn't make it, keep an eye on the `Write the Docs YouTube channel <https://www.youtube.com/channel/UCr019846MitZUEhc6apDdcQ>`__ to at least see the talks.
```
Comment (ericholscher) [inline docs/blog/newsletter-june-2022.rst]: This is out now! https://www.youtube.com/playlist?list=PLZAeFn6dfHpnDhFvXG8GprqlLlzSQRBui

**Review by ericholscher — APPROVED** [review]
Think this is good to merge on my side 👍

File: docs/blog/newsletter-june-2022.rst [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+          </td>
+        </tr>
+      </tbody>
+    </table>
+    <hr>
+
+*Interested in sponsoring the newsletter? Take a look at our* `sponsorship prospectus </sponsorship/newsletter/>`__.
+
+------------------
+Featured job posts
```
Comment (ericholscher) [inline docs/blog/newsletter-june-2022.rst:104]: Just a note that the website shuffles features jobs randomly. I think this is the first time we've had more than 10, but we need to refresh it to get the full list. You can also find the full list in our website sidebar on https://www.writethedocs.org/

Comment (ericholscher) [reply docs/blog/newsletter-june-2022.rst:104]: <ul>
        <li><a href="https://jobs.writethedocs.org/job/743/developer-advocate/">Trunk - Developer Advocate</a></li><li><a href="https://jobs.writethedocs.org/job/736/technical-writer-kotlin/">JetBrains - Technical Writer (Kotlin)</a></li><li><a href="https://jobs.writethedocs.org/job/735/technical-writer-intellij/">JetBrains - Technical Writer (IntelliJ)</a></li><li><a href="https://jobs.writethedocs.org/job/733/technical-writer-software-engineering/">Google - Technical Writer, Software Engineering</a></li><li><a href="https://jobs.writethedocs.org/job/732/technical-writer-cloud-technologies-and-tools/">Google - Technical Writer, Cloud Technologies and Tools</a></li><li><a href="https://jobs.writethedocs.org/job/731/technical-writer-serverless-google-cloud/">Google - Technical Writer, Serverless, Google Cloud</a></li><li><a href="https://jobs.writethedocs.org/job/730/technical-writer-cloud-data-analytics-streams-and-lakes/">Google - Technical Writer, Cloud Data Analytics, Streams and Lakes</a></li><li><a href="https://jobs.writethedocs.org/job/729/senior-manager-technical-writing-developer-acceleration-platforms/">Google - Senior Manager, Technical Writing, Developer Acceleration Platforms</a></li><li><a href="https://jobs.writethedocs.org/job/728/technical-writer-data-analytics-bigquery-google-cloud/">Google - Technical Writer, Data Analytics, BigQuery, Google Cloud</a></li><li><a href="https://jobs.writethedocs.org/job/727/technical-content-strategist-developer-relations-content-google-cloud/">Google - Technical Content Strategist, Developer Relations Content, Google Cloud</a></li><li><a href="https://jobs.writethedocs.org/job/726/developer-evangelist/">Twilio - Developer Evangelist</a></li><li><a href="https://jobs.writethedocs.org/job/725/developer-educator-remote-apj/">Twilio - Developer Educator - Remote APJ</a></li><li><a href="https://jobs.writethedocs.org/job/718/technical-writer/">Wix.com - Technical Writer</a></li><li><a href="https://jobs.writethedocs.org/job/649/technical-writer/">Semaphore - Technical Writer</a></li><li><a href="https://jobs.writethedocs.org/job/717/technical-writer-developer-documentation-fully-remote/">Humanitec GmbH - Technical Writer (Developer Documentation) (fully remote)</a></li><li><a href="https://jobs.writethedocs.org/job/714/product-documentation-manager/">ReadMe - Product Documentation Manager</a></li></ul>

Comment (ericholscher) [reply docs/blog/newsletter-june-2022.rst:104]: I've actually updated the website so it shows all featured jobs 👍

File: docs/blog/newsletter-june-2022.rst (hunk 2 above)
Comment (CollierCZ) [reply docs/blog/newsletter-june-2022.rst]: Thanks for noticing. I think the one in Florida is the same -- an East Coast quorum. I'll try to make that clear

**Pull request merged**